### ISSUE TITLE
Specifying multiple bins picks one at random

### ISSUE THREAD

**Issue by xangelix** [ronnychevalier/cargo-multivers#15]
Specifying multiple bins in a single command, either by having multiple in a package's `Cargo.toml` and building an entire -p, or by specifying multiple bins `--bin`, results in only one being built.

**Comment by ronnychevalier** [comment]
Yes, I never really used it on a project with multiple binaries yet, so I never tried to handle such cases.
But I think it can be fixed easily.

**Comment by ronnychevalier** [comment]
cargo multivers 0.11.1 contains a fix that detects when multiple bins are present in package. If so it returns an error, and suggest to use `cargo multivers -- --bin my_bin` to select a specific bin in a package.
